.travis.yml

```diff
@@ -1,25 +1,29 @@
 language: python
-
 python:
-    - "3.6"
-
+- '3.6'
 branches:
   only:
-    - master
-
+  - master
 services:
-  - docker
-
+- docker
 sudo: required
-
 before_install:
-    - make travis_up
-
+- make travis_up
 script:
-    - make travis_test
-    - make travis_pii_check
-
+- make travis_test
+- make travis_pii_check
 after_success:
-    - pip install -U codecov
-    - docker exec notes /edx/app/notes/notes/.travis/run_coverage.sh
-    - codecov
+- pip install -U codecov
+- docker exec notes /edx/app/notes/notes/.travis/run_coverage.sh
+- codecov
+env:
+  matrix:
+  - DOCKER_USERNAME=edxbuilder
+  global:
+  - secure: UNyJ1U9rQWRrcOabFygKV4doxgS7MWz+TlVU+MftIIzetYRs4wKQUeK4myVbKUL+UR88M4TVhTrsfM1VHYXkdfmFQq0A84HyTd+/iIxf71ikVaSMq6tyr8R6lUsi5/fljYpah5+IX5g1RERS1qTHsP7tfteWJnlJ8noVOY2fhBo=
```

Comment:
You should put the name of the keys that you encrypted here so we find find it easily for rotation purposes.

Reply:
:+1: